okio-files/src/commonMain/kotlin/okio/Filesystem.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2020 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package okio
+
+abstract class Filesystem {
+  /**
+   * The current process' working directory. This is the result of the `getcwd` command on POSIX
+   * and the `user.dir` System property in Java. This is an absolute path that all relative paths
+   * are resolved against for by this filesystem.
+   */
+  abstract val cwd: Path
```

Comment:
Does this reflect changes via chdir?

Reply:
Ooooh I think it should. I don’t do anything off JVM so that’s stuff I’ll need to learn.

Reply:
One thing that’s weird is I could make this a `var` and call `chdir` when you set it!